### ISSUE TITLE
Image brackground 

### ISSUE THREAD

**Issue by delyriand** [magento/magento2#8799]
### Preconditions
1. Magento 2.1.3
2. PHP 7

### Steps to reproduce
1. Edit view.xml file 
`ex: theme-frontend-luma/etc/view.xml`
2. Add background color attribute to an image
```xml
<image id="related_products_list" type="small_image">
     <width>152</width>
     <height>230</height>
     <background>[250,247,247]</background>
</image>
```
3. Clear cache and reload images.

### Expected result
1. Image with background

### Actual result
1. Product image always has a black background
![image](https://cloud.githubusercontent.com/assets/465524/23606432/fd61f83e-0260-11e7-9bcd-f1709d934fbf.png)

------

The problem comes from 2 methods:

- Method  Magento\Catalog\Model\ImageExtractor::processImageBackground : The method return a string array 
- Magento\Framework\Image\Adapter\AbstractAdapter::backgroundColor : If the color is not an integer, the background is ignored.

**Comment by korostii** [comment]
[Relevant part in the dev docs](http://devdocs.magento.com/guides/v2.1/frontend-dev-guide/themes/theme-images.html)

**Comment by ArthurTransoftgroup** [comment]
@delyriand  We've created internal ticket MAGETWO-70383 to address this issue.

**Comment by magento-engcom-team** [comment]
@delyriand, thank you for your report.
We've created internal ticket(s) MAGETWO-70383 to track progress on the issue.

**Comment by raumatbel** [comment]
@magento-engcom-team can you assign me please?

**Comment by magento-team** [comment]
Internal ticket to track issue progress: MAGETWO-83067

**Comment by magento-team** [comment]
Internal ticket to track issue progress: MAGETWO-83066

**Comment by okorshenko** [comment]
The issue has been fixed in 2.2-develop branch and will be available with 2.2.2 release

**Comment by magento-team** [comment]
Internal ticket to track issue progress: MAGETWO-83065
